@here/harp-mapview/lib/Tile.ts

```diff
@@ -702,15 +685,34 @@ export class Tile implements CachedResource {
      * @param decodedTile The decoded tile to set.
      */
     set decodedTile(decodedTile: DecodedTile | undefined) {
-        if (decodedTile && decodedTileHasGeometry(decodedTile)) {
-            if (decodedTile.copyrightHolderIds !== undefined) {
-                this.copyrightInfo = decodedTile.copyrightHolderIds.map(id => ({ id }));
-            }
-            this.setDecodedTile(decodedTile);
-        } else {
-            // empty tiles are traditionally ignored and don't need decode
+        this.m_decodedTile = decodedTile;
+        this.invalidateResourceInfo();
+
+        if (decodedTile === undefined) {
+            return;
+        }
+
+        if (decodedTile.geometries.length === 0) {
             this.forceHasGeometry(true);
         }
+
+        if (decodedTile.boundingBox !== undefined) {
+            // If the decoder provides a more accurate bounding box than the one we computed from
+            // the flat geo box we take it instead.
+            this.boundingBox.copy(decodedTile.boundingBox);
+        }
+        this.invalidateResourceInfo();
```

Comment:
Is this still needed after the call on line 689?

Reply:
ah, true. To much copy paste in this change.